WalletWasabi.Fluent/ViewModels/WalletManagerViewModel.cs

```diff
@@ -35,7 +36,8 @@ public partial class WalletManagerViewModel : ViewModelBase
 		[AutoNotify] private ObservableCollection<WalletViewModelBase> _wallets;
 		[AutoNotify] private bool _anyWalletStarted;
 
-		public WalletManagerViewModel(WalletManager walletManager, UiConfig uiConfig, BitcoinStore bitcoinStore, LegalChecker legalChecker)
+		public WalletManagerViewModel(WalletManager walletManager, UiConfig uiConfig, BitcoinStore bitcoinStore,
+			LegalChecker legalChecker, TransactionBroadcaster transactionBroadcaster)
```

Comment:
Store the TransactionBroadcaster reference and you can pass that at line 275. With this, you don't need to pass the Broadcaster to the wallet classes.

Reply:
@soosr alright